Fix docker so the app waits for mongo

brilvio opened the pull request:
Sorry to hijack and fix the issue, but I had this problem before.

Now looking at the issue it has has nothing to do with waiting for `mongo` to start, but still you can see the implementation here.

It uses this implementation https://github.com/ufoscout/docker-compose-wait/

brilvio committed:
Fixes #27

w3cj commented:
Fails to connect and then kills api container on Mac. Tried specifying some env vars to change the timeout / max failures, but no luck.

brilvio commented:
I will test on linux to see if works and get back to you.

Commit:
:fix: - Changed the version of `docker-compose-wait`
- Changed the CMD on `Dockerfile`

brilvio commented:
> Fails to connect and then kills api container on Mac. Tried specifying some env vars to change the timeout / max failures, but no luck.

It was failing for me as well, it was connecting but then it would exit out of the container.

I updated the version of `docker-compose-wait` and changed the `CMD` line on `Dockerfile` the old way only run the `/wait` command, sorry for not testing I was using windows when I made the changes.

Now I did test on linux Ubuntu and it is working as intended.
